generators/generator-microsoft-bot-adaptive/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": [
+    "generators/**/templates/*"
```

Comment:
Should this be "generators/**/templates/**" to handle for nested directories underneath the templates dir?